### PR TITLE
Wrap definitions for Google benchmark 1.4.1

### PR DESCRIPTION
**Pull request by jeandet**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+lib_benchmark = library('gbenchmark',
+                        benchmark_sources,
+                        include_directories:inc,
+                        cpp_args : '-DNDEBUG',
+                        dependencies : thread_dep)
+
+google_benchmark_dep = declare_dependency(
+                        link_with:lib_benchmark,
+                        include_directories:inc
+                        )
```
Comment (sarum9in) [inline meson.build]: Please make this closing parentheses consistent with the rest of the file.

File: meson.build [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+lib_benchmark_main = library('benchmark-main',
+                        benchmark_main_sources,
+                        include_directories:inc,
+                        cpp_args : '-DNDEBUG',
+                        dependencies : google_benchmark_dep)
+
+google_benchmark_main_dep = declare_dependency(
+                        link_with:lib_benchmark_main,
+                        include_directories:inc
+                        )
```
Comment (sarum9in) [inline meson.build]: Same with parentheses.

**Review by sarum9in — CHANGES_REQUESTED** [review]
```
meson.build:31:0: ERROR:  File src/benchmark_api_internal.cc does not exist.
```

**Comment by jeandet** [comment]
@sarum9in sorry for the mistake, I forgot to test against 1.4.1 branch I was on master...

**Review by sarum9in — APPROVED** [review]

**Pull request merged**